deploy/server.conf

```diff
@@ -21,6 +31,10 @@ server {
     gzip_http_version 1.1;
     gzip_types text/plain text/css application/json application/javascript text/xml;
 
+    location ~ ^/user {
```

Comment:
My regex-fu may be failing me but should this be modified to include `/welsh/user`, too (when it eventually exists)? Don't underestimate the power of a gwadu gwasanaeth wedi'i ddosbarthu attack.

Reply:
Ah good point!